File: 1491.average-salary-excluding-the-minimum-and-maximum-salary/solution.kt
import kotlin.math.*
class Solution {
    fun average(salary: IntArray): Double {
        var lowest: Double = 10000000.0
        var highest: Double = 0.0
        var total: Double = 0.0

        for (s in salary) {
            var sD = s.toDouble()

            lowest = min(lowest, sD)
            highest = max(highest, sD)
            total += sD
        }
        return ((total - lowest - highest) / (salary.count() - 2))
    }
}

File: 1491.average-salary-excluding-the-minimum-and-maximum-salary/solution.py
class Solution:
    def average(self, salary: List[int]) -> float:
        lowest = 10000000
        highest = 0
        total = 0
        for s in salary:
            lowest = min(lowest, s)
            highest = max(highest, s)
            total += s

        return (total - lowest - highest) / (len(salary) - 2)
